src/java/picard/analysis/artifacts/Transition.java

```diff
@@ -51,4 +46,46 @@ public Transition matchingRef() {
     public String toString() {
         return this.ref + ">" + this.call;
     }
+
+    protected enum Base {
+        A ('A'),
+        C ('C'),
+        G ('G'),
+        T ('T');
+
+        public byte base;
+
+        private Base(final char base) {
+            this.base = (byte)base;
+        }
+    }
+
+    static protected final int[] baseIndexMap = new int[256];
```

Comment:
explain that only 4 of the values will be used but that you want to be "allow" an attempted access by any byte/char value.